.github/workflows/ci.yml

```diff
@@ -78,3 +78,19 @@ jobs:
 
       - name: "Coverage"
         run: echo "$(cat ./.phpunit/coverage.txt)"
+
+      - name: Make code coverage badge
+        uses: timkrase/phpunit-coverage-badge@v1.2.1
+        with:
+          coverage_badge_path: .github/coverage.svg
+          # push badge later on
+          push_badge: false
+
+      - name: Git push badges to image-data branch
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          publish_dir: .github
+          publish_branch: image-data
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I have added `PHP_GA4_TOKEN` as secret with token to manage repo. Hope it has the right auths.

Reply:
Sorry for the delay, lots of stuff happening rn.

Reply:
Hello, I've updated the token name.